system/rig_system.go

```diff
@@ -89,10 +161,16 @@ func NewRigSystem(skey int64, MAC, name string, options ...Option) (*RigSystem,
 	}
 	for _, sensor := range sys.Sensors {
 		sensor.Mac = sys.Controller.Mac
+		sys.Controller.Inputs += sensor.Pin + ","
```

Comment:
This will result in a trailing comma after the last entry. We don't want that. Ditto below.